backend/EasyMeets.Core/EasyMeets.Core.BLL/Services/UserService.cs

```diff
@@ -13,9 +12,13 @@ public class UserService : BaseService, IUserService
         public UserService(EasyMeetsCoreContext context, IMapper mapper) : base(context, mapper)
         { }
 
-        public async Task<UserDto> GetUserPreferences(long userId)
+        public async Task<UserDto?> GetUserPreferences(long userId)
         {
             var user = await _context.Users.FirstOrDefaultAsync(_ => _.Id == userId);
+            if (user is null)
+            {
+                return default;
+            }
```

Comment:
Why default? It would be better to throw exception

Reply:
done